doc/src/devdocs/ast.md

```diff
@@ -1,480 +1,440 @@
+  * `the_exception`
 
-``the_exception``
-    yields the caught exception inside a ``catch`` block. This is the value of
-    the run time system variable ``jl_exception_in_transit``.
+    yields the caught exception inside a `catch` block. This is the value of the run time system variable
+    `jl_exception_in_transit`.
+  * `enter`
 
-``enter``
-    enters an exception handler (``setjmp``). ``args[1]`` is the label of the
-    catch block to jump to on error.
+    enters an exception handler (`setjmp`). `args[1]` is the label of the catch block to jump to on
+    error.
+  * `leave`
 
-``leave``
-    pop exception handlers. ``args[1]`` is the number of handlers to pop.
+    pop exception handlers. `args[1]` is the number of handlers to pop.
+  * `inbounds`
 
-``inbounds``
-    controls turning bounds checks on or off. A stack is maintained; if the
-    first argument of this expression is true or false (``true`` means bounds
-    checks are disabled), it is pushed onto the stack. If the first argument is
-    ``:pop``, the stack is popped.
+    controls turning bounds checks on or off. A stack is maintained; if the first argument of this
+    expression is true or false (`true` means bounds checks are disabled), it is pushed onto the stack.
+    If the first argument is `:pop`, the stack is popped.
+  * `boundscheck`
 
-``boundscheck``
-    indicates the beginning or end of a section of code that performs a bounds
-    check. Like ``inbounds``, a stack is maintained, and the second argument
-    can be one of: ``true``, ``false``, or ``:pop``.
+    indicates the beginning or end of a section of code that performs a bounds check. Like `inbounds`,
+    a stack is maintained, and the second argument can be one of: `true`, `false`, or `:pop`.
+  * `copyast`
 
-``copyast``
-    part of the implementation of quasi-quote. The argument is a surface syntax
-    AST that is simply copied recursively and returned at run time.
+    part of the implementation of quasi-quote. The argument is a surface syntax AST that is simply
+    copied recursively and returned at run time.
+  * `meta`
 
-``meta``
-    metadata. ``args[1]`` is typically a symbol specifying the kind of metadata,
-    and the rest of the arguments are free-form. The following kinds of metadata
-    are commonly used:
+    metadata. `args[1]` is typically a symbol specifying the kind of metadata, and the rest of the
+    arguments are free-form. The following kinds of metadata are commonly used:
 
-    ``:inline`` and ``:noinline``: Inlining hints.
+    `:inline` and `:noinline`: Inlining hints.
 
-    ``:push_loc``: enters a sequence of statements from a specified source location.
-      - ``args[2]`` specifies a filename, as a symbol.
-      - ``args[3]`` optionally specifies the name of an (inlined) function that originally contained the code.
+      * `:push_loc`: enters a sequence of statements from a specified source location.
```

Comment:
this didn't previously have a bullet, though the `inline` and `pop_loc` lines probably should instead of the way it was